internal/form/service.go

```diff
@@ -0,0 +1,113 @@
+package form
+
+import (
+	"context"
+
+	databaseutil "github.com/NYCU-SDC/summer/pkg/database"
+	logutil "github.com/NYCU-SDC/summer/pkg/log"
+	"github.com/google/uuid"
+	"github.com/jackc/pgx/v5/pgtype"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/trace"
+	"go.uber.org/zap"
+)
+
+type Querier interface {
+	Create(ctx context.Context, params CreateParams) (Form, error)
+	Update(ctx context.Context, params UpdateParams) (Form, error)
+	Delete(ctx context.Context, id uuid.UUID) error
+	GetByID(ctx context.Context, id uuid.UUID) (Form, error)
+	List(ctx context.Context) ([]Form, error)
+}
+
+type Service struct {
+	logger  *zap.Logger
+	queries Querier
+	tracer  trace.Tracer
+}
+
+func NewService(logger *zap.Logger, db DBTX) *Service {
+	return &Service{
+		logger:  logger,
+		queries: New(db),
+		tracer:  otel.Tracer("forms/service"),
+	}
+}
+
+func (s *Service) Create(ctx context.Context, req FormRequest, userID uuid.UUID) (Form, error) {
+	ctx, span := s.tracer.Start(ctx, "CreateForm")
```

Comment:
The tracer already has a namespace defined (otel.Tracer("forms/service") on line 33), so there's no need to repeat "Form" in the span name here.